src/helpers.py

```diff
@@ -35,6 +36,29 @@
 # You can keep this number low, maybe 60 seconds.
 WATCH_CLIENT_TIMEOUT = os.environ.get("WATCH_CLIENT_TIMEOUT", 66)
 
+
+def get_logger(name):
+    """
+    Instantiate a logger with the specified name, ensuring the LOG_LEVEL environment variable is valid.
+
+    :param name: logger name
+    :return: a logger instance
+    """
+    log = logging.getLogger(name)
+    try:
+        log_level = os.getenv("LOG_LEVEL", logging.INFO)
+        log_level = log_level.upper() if isinstance(log_level, str) else log_level
+        log.setLevel(log_level)
+        return log
+    except (ValueError, TypeError) as e:
+        log.error(f"Unable to set log level: {e}")
+        exit(1)
```

Comment:
I would prefer not to exit here. It just falls back to INFO (default) level and can run just fine.

Reply:
make sense. In this case I would define the log as a warning:

```
log.warning("Unable to set log level: {e}. Define default level to INFO")
```